src/overlays/actors/ovl_En_Sekihi/z_en_sekihi.c

```diff
@@ -60,7 +60,7 @@ void EnSekihi_Init(Actor* thisx, PlayState* play) {
     }
 
     if ((params == SEKIHI_TYPE_4) && (((gSaveContext.save.skullTokenCount & 0xFFFF)) >= 30)) {
```

Comment:
```suggestion
    if ((params == SEKIHI_TYPE_4) && (((gSaveContext.save.skullTokenCount & 0xFFFF)) >= SPIDER_HOUSE_TOKENS_REQUIRED)) {
```
Would be nice to setup getter/setter macros for `skullTokenCount` soon, although not in this PR